src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/PyrisJobService.java

```diff
@@ -0,0 +1,124 @@
+package de.tum.in.www1.artemis.service.connectors.pyris;
+
+import java.security.SecureRandom;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+
+import de.tum.in.www1.artemis.service.connectors.pyris.job.PyrisJob;
+import de.tum.in.www1.artemis.web.rest.errors.AccessForbiddenException;
+
+@Service
+@Profile("iris")
+public class PyrisJobService {
+
+    private final HazelcastInstance hazelcastInstance;
+
+    private IMap<String, PyrisJob> jobMap;
+
+    @Value("${server.url}")
+    private String serverUrl;
+
+    @Value("${eureka.instance.instanceId:unknown}")
+    private String instanceId;
+
+    @Value("${artemis.iris.jobs.timeout:300}")
+    private int jobTimeout; // in seconds
+
+    public PyrisJobService(HazelcastInstance hazelcastInstance) {
+        this.hazelcastInstance = hazelcastInstance;
+    }
+
+    /**
+     * Add listeners for build job queue changes.
```

Comment:
looks like a copy paste mistake, please fix it or explain why you would be interested in the build job queue